content/docs/refs-and-the-dom.md

```diff
@@ -221,25 +213,22 @@ class CustomTextInput extends React.Component {
     };
 
     this.focusTextInput = () => {
-      // Focus the text input using the raw DOM API
+      // Foca o input de texto usando a API DOM diretamente
       if (this.textInput) this.textInput.focus();
     };
   }
 
   componentDidMount() {
-    // autofocus the input on mount
+    // auto-foca o input na montagem
     this.focusTextInput();
   }
 
   render() {
-    // Use the `ref` callback to store a reference to the text input DOM
-    // element in an instance field (for example, this.textInput).
+    // Utiliza a `ref` com callback para armazenar uma referência ao elemento DOM
+    // do input de texto em um campo de instância (por exemplo, this.textInput)
     return (
       <div>
-        <input
-          type="text"
-          ref={this.setTextInputRef}
-        />
+        <input type="text" ref={this.setTextInputRef} />
```

Comment:
```suggestion
        <input
          type="text"
          ref={this.setTextInputRef}
        />
```

Tb acho que o editor tá com um auto-format, se for o caso, to marcando os pontos levantados pelo @cezaraugusto aqui que você pode commitar diretamente pela interface aqui do GitHub

Reply:
Sim! Estou fazendo isso. Eu realmente não tinha percebido que o editor estava fazendo a correção automática. Foi mal!